src/openklant/components/klanten/api/filters.py

```diff
@@ -18,6 +22,31 @@ class KlantFilter(FilterSet):
         field_name="natuurlijk_persoon__inp_a_nummer",
         help_text=get_help_text("klanten.NatuurlijkPersoon", "inp_a_nummer"),
     )
+    subject_natuurlijk_persoon__geboortedatum = filters.CharFilter(
+        field_name="natuurlijk_persoon__geboortedatum",
+        help_text=mark_oas_difference(_("Geboortedatum van de persoon")),
+        lookup_expr="exact",
+    )
+    subject_natuurlijk_persoon__geboortedatum__gt = filters.CharFilter(
+        field_name="natuurlijk_persoon__geboortedatum",
+        help_text=mark_oas_difference(_("Geboortedatum van de persoon")),
+        lookup_expr="gt",
+    )
+    subject_natuurlijk_persoon__geboortedatum__gte = filters.CharFilter(
+        field_name="natuurlijk_persoon__geboortedatum",
+        help_text=mark_oas_difference(_("Geboortedatum van de persoon")),
+        lookup_expr="gte",
+    )
+    subject_natuurlijk_persoon__geboortedatum__lt = filters.CharFilter(
+        field_name="natuurlijk_persoon__geboortedatum",
+        help_text=mark_oas_difference(_("Geboortedatum van de persoon")),
+        lookup_expr="lt",
+    )
+    subject_natuurlijk_persoon__geboortedatum__lte = filters.CharFilter(
+        field_name="natuurlijk_persoon__geboortedatum",
+        help_text=mark_oas_difference(_("Geboortedatum van de persoon")),
+        lookup_expr="lte",
+    )
```

Comment:
Blijkbaar kun je niet meer 1 filter aanmaken met een lijst als `lookup_expr`. Ik heb het ook geprobeerd via `fields` in de `Meta`, maar dan is de naam niet gelijk aan de andere filters voor natuurlijke personen (`natuurlijk_persoon__geboortedatum` i.p.v. `subject_natuurlijk_persoon__geboortedatum`)